### PR TITLE
Ignore down interfaces when calculating host MTU.

### PR DESCRIPTION
**Pull request by fasaxc**
## Description

Ignore interfaces that are down when auto-detecting MTU.
## Todos

- [x] Tests
- [ ] Documentation
- [x] Release note

## Release Note

```release-note
Host MTU auto-detection now ignores interfaces that are down.
```

## Reminder for the reviewer

Make sure that this PR has the correct labels and milestone set.
Host MTUEvery PR needs one `docs-*` label.

- `docs-pr-required`: This change requires a change to the documentation that has not been completed yet.
- `docs-completed`: This change has all necessary documentation completed.
- `docs-not-required`: This change has no user-facing impact and requires no docs.

Every PR needs one `release-note-*` label.

- `release-note-required`: This PR has user-facing changes. Most PRs should have this label.
- `release-note-not-required`: This PR has no user-facing changes.

Other optional labels:

- `cherry-pick-candidate`: This PR should be cherry-picked to an earlier release. For bug fixes only.
- `needs-operator-pr`: This PR is related to install and requires a corresponding change to the operator.

### CODE REVIEW

File: felix/fv/mtu_test.go
Comment (song-jiang) [inline felix/fv/mtu_test.go:47]: Does `ContainSubstring` succeed if output is `13508` but the correct value of mtu should be `1350`?

**Review by song-jiang — COMMENTED** [review]
Looking good! Just one comment.

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, squash the commits when I merge it, and leave the branch after I've merged it.

**Review by song-jiang — APPROVED** [review]
LGTM

**Pull request merged**
